server/src/main/java/org/apache/druid/server/coordinator/duty/KillCompactionConfig.java

```diff
@@ -133,6 +133,7 @@ private int tryDeleteCompactionConfigs() throws RetryableException
           return updated;
         },
         new AuditInfo(
+            "KillCompactionConfig",
             "KillCompactionConfig",
```

Comment:
identity and author are KillCompactionConfig?

Reply:
Thanks for catching this. Since we are updating auditing, I guess it would be better to use `druid_system` or something as the author instead.